### DISCUSSION TITLE
ai code completion other than co-pilot

### DISCUSSION THREAD

**Discussion by syahbes** [zed-industries/zed#14345] [Feature Requests]
Hi :) I'm a linux user so this week was my first time using Zed and I love it !
thanks !!

is there an option to config other 'ai code completion' like Tabnine (https://www.tabnine.com) or Codeium (https://codeium.com)

**Comment by tepavcevic** [comment]
There is option to have [Supermaven](https://supermaven.com/), that has a free tier and is pretty fast, so take a look at it.
You can enable it in the settings:
```json
"features": {
    "inline_completion_provider": "supermaven"
  }
```

**Comment by ghost** [comment]
hopefully Ollama soon

**Comment by ghost** [comment]
Yeah, Ollama would be awesome
